cesium/writer_stream.go

```diff
@@ -472,7 +472,15 @@ func (w *idxWriter) updateHighWater(s telem.Series) error {
 			s.DataType,
 		)
 	}
+	if s.Len() == 0 {
```

Comment:
Why should this error rather than doing nothing and just return?

Reply:
Agreed, we definitely shouldn't error here. If this series has length 0, then all other series in the frame must have length 0 per `validateWrite` so no data gets written. It honestly might be clearer to do this check just after the call to `validateWrite` on line 340 though. It feels more explicit that silently writing no data to all unary writers.

Reply:
If we do choose to keep it here, we should add a test case that asserts its ok to write an empty series to a unary DB

Reply:
So if the series is empty do we still want to be able to write? Then would it make more sense just to add to the condition on line 351 to not update the watermark if the series is empty?